js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,44 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const areNumbers = (args) => args.every(isNumber);
+
+function* getRange(start, end) {
+  let temp = start;
+  while (temp <= end) {
```

Comment:
It is really important to consider whether whether you want to include the last element as you did here, to not include it. For the record, python's range does not include the last element.

Reply:
yes, right.
current scope of sum-all specs says to return sum of all numbers between them, including them.`getRange` currently works according to that.

If `getRange` will be grown further. it should be .
Either return numbers between `start` and `end`. (Note: Excluding `start` as well unlike Python one).
OR return numbers between `start` and `end`, including parameters passed.

What say?